### ISSUE TITLE
Add explanatory section to the header of the prototype

### ISSUE THREAD

**Issue by JonathanReeve** [open-editions/corpus-joyce-ulysses-tei#26]
If anyone should land on [the prototype page](http://ulysses-tei.netlify.com/), it'd be helpful to explain what it is, and link to the repository.

**Comment by yellwork** [comment]
Good point. I'm happy to draft something if you want to handle the XSLT to get it into the HTML file?

**Comment by JonathanReeve** [comment]
Sounds good!

**Comment by cOdEeInStEiN123** [comment]
So sorry to jump in like this .. am very new to contributing .. can anyone direct me to some kind of a guide on how to help??? Thank you

**Comment by JonathanReeve** [comment]
@cOdEeInStEiN123, welcome! The [guide you want is here](https://open-editions.org/contributing/). Let me know if you have any questions.

**Comment by spykelionel** [comment]
Heloo!, What's the explanatory text we have here?

**Comment by JonathanReeve** [comment]
The idea is to explain what the prototype does, and what the project is doing, so that if anyone lands on the page, they'll be able to understand what they're seeing. So in order to write this, you should really know all you can about the project, its aims, and the aims of this particular edition.

**Comment by JonathanReeve** [comment]
This may not be the easiest, or best, issue for a first-time contributor. Maybe you can add some markup to the text, instead?

**Comment by Yekrats** [comment]
@JonathanReeve  I found this issue via the link at [Works Hub](https://functional.works-hub.com/issue/869c2516-8d17-4982-b222-561134f10d07). Is this still an issue that you want resolved? Right now, the prototype link you have above goes to a 404 page. (Which is probably something that you also need fixed!)

> > > > > > <img width="464" height="303" alt="Image" src="https://github.com/user-attachments/assets/d72189ca-a590-4418-be9e-958fa1853a6f" />

Are one of these issues something that you want help on? If so, I can probably help. I have a real-life pre-A.I. writing diploma gathering dust in addition to my computer diploma.

**Comment by JonathanReeve** [comment]
Hi @Yekrats . Thanks for the heads-up. It looks like Netlify deprecated the build servers I was using, so that, even though it was a static site, it failed to serve the HTML. It should be fixed now. Please do feel free to tackle this issue, if it interests you. The same goes for anything with the Open Editions repos.
